apps/ledger-live-desktop/tests/page/modal/send.modal.ts

```diff
@@ -34,10 +37,20 @@ export class SendModal extends Modal {
 
   @step("Enter recipient as $0")
   async fillRecipient(recipient: string) {
+    if (!recipient) {
+      throw new Error("Recipient is not provided");
+    }
```

Comment:
you don't need to check if (!recipient) because TypeScript guarantees that recipient will always be a string